README.md

```diff
@@ -27,7 +27,7 @@ Read more on [ui2.nuxt.com](https://ui2.nuxt.com)
 ## Installation
 
 ```bash
-npx nuxi@latest module add ui
+npx nuxi@latest module add ui@2
```

Comment:
Isn't it supposed to be `ui2`? 🤔 https://github.com/nuxt/starter/blob/templates/templates/ui2.json

```suggestion
npx nuxi@latest module add ui2
```

Reply:
For me, @benjamincanac, it's not a template but a module definition. 😊

However, there is still an issue with this command. I initially thought this MR had fixed it, but it turns out you still need to specify a full version with this command: https://github.com/nuxt/cli/pull/702.

Reply:
Oh yes I got confused my bad 😬